locales/fr-FR.json

```diff
@@ -50,6 +50,9 @@
   "needsPrettier": {
     "message": "Ajouter Prettier pour le formatage du code\u00a0?"
   },
+  "needsVueDevtools": {
+    "message": "Add VueDevtools extension for debugging? (experimental)"
```

Comment:
French version can be: `Ajouter l'extension Vue DevTools pour le débogage\u00a0? (expérimental)`

Reply:
Thanks! I will add it.